skimage/future/__init__.py

```diff
@@ -5,6 +5,6 @@
 production code that will depend on updated skimage versions.
 """
 
-from . import graph
+from . import graph, objdetect
 
-__all__ = ['graph']
+__all__ = ['graph', 'objdetect']
```

Comment:
I agree with @JDWarner, go with `detect`. It mirrors most of our API, with some well-justified exceptions (`filters`) and others not so well justified, that we might consider updating for 1.0: `segment`, `restore`, `view`.